workspace/apps/pidp/src/app/features/profile/pages/college-licence/college-licence-information/college-licence-information.page.ts

```diff
@@ -68,12 +68,25 @@ export class CollegeLicenceInformationPage implements OnInit {
     this.title = this.route.snapshot.data.title;
   }
 
+  public scrollToTop(): Promise<void> {
+    return new Promise<void>((resolve) => {
+      window.scrollTo({ top: 0 });
+      setTimeout(() => {
+        resolve();
+      }, 300);
```

Comment:
I think we can define the timeout value in the constants.

Reply:
This function can be replaced as it exists as a service